playbooks/common/openshift-node/config.yml

```diff
@@ -159,7 +159,6 @@
     openshift_nodes: "{{ hostvars
                          | oo_select_keys(groups['oo_nodes_to_config'])
                          | oo_collect('openshift.common.hostname') }}"
-    openshift_node_vars: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config']) }}"
```

Comment:
After seeing the visual effects of this, I don't think there are enough `+`'s for how much I want to :+1:  this.